mne/viz/misc.py

```diff
@@ -769,12 +765,12 @@ def plot_filter(h, sfreq, freq=None, gain=None, title=None, color='#1f77b4',
     axes[1].set(ylabel='Magnitude (dB)', xlabel='', xscale=fscale)
     sl = slice(0 if fscale == 'linear' else 1, None, None)
     axes[2].plot(f[sl], gd[sl], color=color, linewidth=2, zorder=4)
-    axes[2].set(xlim=flim, ylabel='Group delay (sec)', xlabel='Frequency (Hz)',
+    axes[2].set(xlim=flim, ylabel='Group delay (s)', xlabel='Frequency (Hz)',
```

Comment:
Maybe it would be nicer if you did `fig, (time_ax, freq_ax, delay_ax) = plt.subplots(3, squeeze=True)` in 754.